test/Faker/Provider/fr_FR/PhoneNumberTest.php

```diff
@@ -58,4 +58,24 @@ public function testMobileNumber07WithSeparator()
             $i++;
         }
     }
+    
+    public function testServiceNumber()
+    {
+        do {
```

Comment:
I'd prefer if you tested `phoneNumber08WithSeparator` instead, that would avoid doing the loop